src/main/java/Duke.java

```diff
@@ -1,10 +1,62 @@
+import java.util.Scanner;
+
 public class Duke {
+    // Variables
+    static final String HORIZ_LINE = "____________________________________________________________";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Task[] tasks = new Task[100];
+
+        // Init
+        System.out.println("Hello! I'm Jarvis");
+        System.out.println("What can I do for you?");
+        System.out.println(HORIZ_LINE);
+
+        // Read user input
+        Scanner userInput = new Scanner(System.in);
+        String inputTxt = userInput.nextLine();
+        for (int i = 0; i < tasks.length; i++) {
+            if (inputTxt.equals("bye")) {
+                break;
+            } else if (inputTxt.startsWith("done")) {
+                int idx = Integer.parseInt(inputTxt.split(" ")[1]) - 1;
+                if (idx < i) {
+                    tasks[idx].setDone();
+                    System.out.println("Nice! I've marked this as done:");
+                    System.out.println(tasks[idx].getTask());
+                } else {
+                    System.out.println("Task out of limit");
+                }
+                i--;
+            } else if (inputTxt.equals("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int j = 0; j < i; j++) {
+                    System.out.println((j + 1) + ". " + tasks[j].getTask());
+                }
+                i--;
+            } else if (inputTxt.startsWith("todo")) {
+                tasks[i] = new Todo(inputTxt.substring(5));
+                System.out.println("added: " + tasks[i].getTask());
+            } else if (inputTxt.startsWith("deadline")) {
+                String[] deadlineSplit = inputTxt.substring(9).split("/by");
+                tasks[i] = new Deadline(deadlineSplit[0].trim(), deadlineSplit[1].trim());
+                System.out.println("added: " + tasks[i].getTask());
+            } else if (inputTxt.startsWith("event")) {
+                String[] eventSplit = inputTxt.substring(6).split("/at");
+                tasks[i] = new Event(eventSplit[0].trim(), eventSplit[1].trim());
+                System.out.println("added: " + tasks[i].getTask());
+            } else {
+                System.out.println("Invalid input");
+                i--;
+            }
+
+            System.out.println("Total tasks: " + (i + 1));
+            System.out.println(HORIZ_LINE);
+            inputTxt = userInput.nextLine();
+        }
+
+        // Finalise
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println(HORIZ_LINE);
     }
```

Comment:
The open Scanner resource should be closed after use? Maybe try-with-resource method Java is better in such cases?

Reply:
Great idea, will consider applying that